src/com/codebusters/game/Game.java

```diff
@@ -6,136 +6,160 @@
-////   //     String chapterName = currentChapter.getChapterName();
-////  //      return chapterName.equals("End") || chapterName.equals("Death");
-////    }
-//
-//    private void updateInventory() {
-//        for (Items item: parser.getItemsToAdd()){
-//            addItemToInventory(item);
-//        }
-//
-//        for (Items item: parser.getItemsToRemove()){
-//            removeItemFromInventory(item);
-//        }
-//    }
-//
-//    private void promptAgain() {
-//        // get the current scene text
-//        String tryAgain = currentGame.getSceneText();
-//        // create message that there was an invalid input
-//        String invalid = "That is an unrecognized input, please try again. Only two word commands are allowed in the form of verb + noun.";
-//
-//        // check if current scene already has the message, if so,
-//        // we don't need to add again
-//        if (!tryAgain.contains(invalid)){
-//            tryAgain += "\n\n";
-//            tryAgain += invalid;
-//        }
-//        currentGame.setSceneText(tryAgain);
-//    }
-//
-//    private void updateGameState(Chapter currentChapter) {
-//        currentGame.setSceneText(currentChapter.getSceneText());
-//        currentGame.setInventory(inventory);
-//    }
-//
-//    //***************INVENTORY ACCESSOR METHODS***************
-//    public boolean findItemInInventory(Items toFind){
-//        for (Items item: inventory){
-//            if (item.getName() == toFind.getName()){
-//                return true;
-//            }
-//        }
-//        return false;
-//    }
-//
-//    private void addItemToInventory(Items toAdd){
-//        // if the item is already in the inventory
-//        if (findItemInInventory(toAdd)){
-//            // then we find the item
-//            for (Items item: inventory){
-//                if (item.getName() == toAdd.getName()){
-//                    // and just add to its count
-//                    item.setCount(item.getCount() + toAdd.getCount());
-//                }
-//            }
-//
-//        // else we just add the item to the inventory list
-//        }else{
-//            inventory.add(toAdd);
-//        }
-//    }
-//
-//    private boolean removeItemFromInventory(Items toLose){
-//        // find the item in the inventory
-//        for (Items item: inventory){
-//            if (item.getName() == toLose.getName()){
-//                // if there is more quantity than the player loses, just update count
-//                if (item.getCount() > toLose.getCount()){
-//                    item.setCount(item.getCount() - toLose.getCount());
-//                // else remove item completely
-//                }else{
-//                    inventory.remove(item);
-//                }
-//                return true;
-//            }
-//        }
-//
-//        // return false if item not found in inventory
-//        return false;
-//    }
+    public GameState currentGame;
```

Comment:
Re-examine these class variables to set their access level (private, public, or protected)